generators/app/templates/gulp/_building.js

```diff
@@ -54,8 +75,31 @@ gulp.task('build-app', ['clean', 'inject-all'], function () {
   return stream;
 });
 
-// copy templates
-gulp.task('build-templates', ['clean'], function () {
+<% if (answers.jade) { -%>
+//compile jade
+gulp.task('jade',['clean-templates'], function (done) {
+     gulp.src(paths.jade)
+      .pipe(jade())
+      .pipe(gulp.dest('.tmp'))
+      .on('end', done);
+});
+
+// copy jade templates to build
+gulp.task('build-templates', ['jade'], function () {
+  return gulp.src('.tmp/**/*.html')
+  .pipe($.if(options.minify, $.htmlmin({
+    removeComments: true,
+    removeCommentsFromCDATA: true,
+    collapseWhitespace: true,
+    conservativeCollapse: true,
+    collapseInlineTagWhitespace: true
+  })))
+  .pipe(gulp.dest(paths.dist));
+});
+<% } else { -%>
+
+// copy html templates to build
+gulp.task('build-templates', ['clean-templates'], function () {
```

Comment:
gulp tasks: please leave the original gulp tasks in tact, I assume that some of the gulp tasks break because of these changes.